frontend/package.json

```diff
@@ -70,8 +71,13 @@
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-plugin-react-refresh": "^0.4.10",
+    "fs": "^0.0.1-security",
```

Comment:
This package and path should not be installed, they are not the packages you think they are. Fs and path are built into Node/Bun.